airqo-platform/AirQo-frontend
# [NetManager] D3 integration part 2

#### Summary of Changes (What does this PR do?)
- Add bar charts
- Add Multilayer pie chart

#### Is this change ready to hit production in its current state?

#### Status of maturity (all need to be checked before merging):


#### How should this be manually tested?
* Fetch and checkout this backend [PR](https://github.com/airqo-platform/AirQo-api/pull/708) branch locally if not already merged to staging.
* Fetch and checkout to this branch locally
* Install requirements `npm install`
* When running this with a locally running `analytics` backend. In the `.env` file , set the `REACT_APP_BASE_ANALYTICS_URL` to point to the local analytics app `http://localhost:5000/api/v1/`
* Start netmanager application `npm run stage-mac` (macOS) or `npm run stage-pc` (windows Platform)
* Confirm the summary of changes listed above

#### What are the relevant tickets?
- [PLAT-891](https://airqoteam.atlassian.net/browse/PLAT-891)
